packages/nimble-components/src/rich-text-editor/template.ts

```diff
@@ -6,19 +6,36 @@ import { iconBoldBTag } from '../icons/bold-b';
 import { iconItalicITag } from '../icons/italic-i';
 import { iconListTag } from '../icons/list';
 import { iconNumberListTag } from '../icons/number-list';
+import { errorTextTemplate } from '../patterns/error/template';
+import { iconExclamationMarkTag } from '../icons/exclamation-mark';
 
 // prettier-ignore
 export const template = html<RichTextEditor>`
-    <template>
+    <template
+        aria-disabled="${x => x.disabled}"
+    >
         <div class="container">
-            <section ${ref('editorContainer')} class="editor-container">
+            <section
+                ${ref('editorContainer')}
+                class="editor-container"
+            >
             </section>
-            <section class="footer-section" part="footer-section">
-                <${toolbarTag}>
+            <${iconExclamationMarkTag}
+                severity="error"
+                class="error-icon ${x => (x.scrollbarWidth >= 0 ? 'scrollbar-width-calculated' : '')}"
```

Comment:
`scrollbar-width-calculated` needs to be an attribute instead of a class which will conflict with user defined class names and for other reasons stated in [architectural philosophies in contributing document](https://github.com/ni/nimble/blob/51b3fa5759b35d653ad3866535c726399c89711b/packages/nimble-components/CONTRIBUTING.md#why-attributes-over-classes)

Consider having this as an attribute and make necessary selector changes in `styles.ts`

Reply:
Suggestion - As per [attribute naming conventions](https://github.com/ni/nimble/blob/51b3fa5759b35d653ad3866535c726399c89711b/packages/nimble-components/CONTRIBUTING.md#attribute-common-name-patterns), consider naming it as `scrollbar-visible`. 

Open for alternative names.

Reply:
Thanks for the references, I have updated it to an attribute instead of a class and named the attribute as `scrollbar-visible` as suggested for the error icon.

Reply:
There is no need to utilize the `scrollbar-visible` condition, as the icon positions itself precisely where we intended. I verified that the UI glitch shown in [this comment for text area](https://github.com/ni/nimble/pull/1119#discussion_r1154889610) is not reproducible for the editor. So I removed this condition from here.